src/magic_folder/api_cli.py

```diff
@@ -150,6 +209,8 @@ def get_client(self):
     subCommands = [
         ["add-snapshot", None, AddSnapshotOptions, "Add a Snapshot of a file to a magic-folder."],
         ["dump-state", None, DumpStateOptions, "Dump the local state of a magic-folder."],
+        ["add-participant", None, AddParticipantOptions, "Add a Participant to a magic-folder."],
+        ["list-participants", None, ListParticipantsOptions, "List all Participants in a magic-folder."],
```

Comment:
@exarkun and I discussed this: the `magic-folder` CLI is a user-facing interface and often needs more consideration of its shape, options, etc. We decided it would be good to have a "low level" CLI that essentially just wraps the available HTTP APIs .. that is, you *could* do basically everything in the `magic-folder-api` with a properly-crafted `curl` command. This API needs fewer considerations around usability etc since it "just" represents the available endpoints.

So, it's more of a convenience for doing debugging or other low-level stuff where you just want to call the API endpoints. I guess we probably both were thinking of the `tahoe *` CLI which has an overwhelming number of sub-commands and options (although I'm not sure that part was made explicit).

We ultimately do want some high-level (that is `magic-folder *`) commands for user-management but those will most likely be centered around "invites" (e.g. see https://github.com/LeastAuthority/magic-folder/pull/312) and we definitely don't want users messing around with `Personal DMD` capability-strings if we can avoid it. (IMO). That said, `list` probably does make sense higher-level but for this PR I just wanted "plumbing" commands to poke at the API (and, arguably / ideally, the whole CLI part probably should have been a separate PR anyway).